code-examples/protect-page-login/react/src/App.tsx

```diff
@@ -40,11 +40,12 @@ function App() {
           setLogoutUrl(data.logout_url)
         })
       })
-      .catch(() => {
+      .catch((err) => {
+        console.error(err)
         // Redirect to login page
         window.location.replace(`${basePath}/ui/login`)
       })
-  })
+  }, [])
```

Comment:
fyi @Benehiko @flrnd without this the hook kept reloading :)

Reply:
woops! :sweat_smile:

Reply:
Yup, that's another good one, missing the dependency array :1st_place_medal: